components/org.wso2.carbon.event.outbound.adapter.websubhub/src/main/java/org/wso2/carbon/event/outbound/adapter/websubhub/model/SecurityEventTokenPayload.java

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright (c) 2022, WSO2 LLC. (http://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.event.outbound.adapter.websubhub.model;
+
+import java.util.Map;
+
+/**
+ * Model class for Security Event Token Payload.
+ */
+public class SecurityEventTokenPayload {
+
+    private String iss;
+    private String jti;
+    private long iat;
+    private String aud;
+    private Map<String, EventPayload> event;
```

Comment:
in this map, can we send multiple event payloads?

Reply:
yes. With different keys (event uris) we can send multiple payloads.